sections/image-with-text.liquid

```diff
@@ -50,7 +50,7 @@
     <div class="image-with-text__text-item grid__item">
       <div
         id="ImageWithText--{{ section.id }}"
-        class="image-with-text__content image-with-text__content--{{ section.settings.desktop_content_position }} image-with-text__content--desktop-{{ section.settings.desktop_content_alignment }} image-with-text__content--mobile-{{ section.settings.mobile_content_alignment }} image-with-text__content--{{ section.settings.height }} gradient color-{{ section.settings.color_scheme }} content-container"
+        class="image-with-text__content image-with-text__content--{{ section.settings.desktop_content_position }} image-with-text__content--desktop-{{ section.settings.desktop_content_alignment }} image-with-text__content--mobile-{{ section.settings.mobile_content_alignment }} image-with-text__content--{{ section.settings.height }} gradient color-{{ section.settings.color_scheme }} content-container{% if settings.animations_reveal_on_scroll and section.settings.content_layout == 'overlap' %} scroll-trigger animate--slide-in{% endif %}"
```

Comment:
This looks ok but it's not exactly the same effect. What it's doing here:
1. The parent `image-with-text` div is sliding in when it comes into viewport
2. The `image-with-text__content` div is sliding in when it comes into the viewport. This is usually after the parent because it's further down the page.

If you want to be more precise about the animations:
1. Move the `scroll-trigger` and `animate--slide-in` class to `image-with-text__media-item` `and `image-with-text__text-item`
2. Add `data-cascade` to each div if `section.settings.content_layout == 'overlap'`